src/playbooks/with_build/vsp_postupgrade_health.yml

```diff
@@ -1,127 +1,11 @@
-        nuage_upgrade: true
-
-    - name: Check if prereq satisfied for upgrade
-      include_role:
-        name: common
-        tasks_from: check-upgrade-prereq
-  vars:
-    report_filename: vstat_postupgrade_health_report
-    vstat_username: "{{ vstat_custom_username | default(vstat_default_username) }}"
-    vstat_password: "{{ vstat_custom_password | default(vstat_default_password) }}"
-  roles:
-    - vstat-health
-  remote_user: "{{ vstat_default_username }}"
-
-- hosts: vcins
-  gather_facts: no
-  serial: 1
-  pre_tasks:
-    - name: Set upgrade flag
-      set_fact:
-        nuage_upgrade: true
-
-    - name: Check if prereq satisfied for upgrade
-      include_role:
-        name: common
-        tasks_from: check-upgrade-prereq
-  vars:
-    report_filename: vcin_postupgrade_health_report
-    vsd_username: "{{ vsd_custom_username | default(vsd_default_username) }}"
-    vsd_password: "{{ vsd_custom_password | default(vsd_default_password) }}"
-  roles:
-    - vcin-health
-
-- hosts: vscs
-  gather_facts: no
-  serial: 1
-  connection: local
-  pre_tasks:
-    - name: Set upgrade flag
-      set_fact:
-        nuage_upgrade: true
-
-    - name: Check if prereq satisfied for upgrade
-      include_role:
-        name: common
-        tasks_from: check-upgrade-prereq
-  vars:
-    report_filename: vsp_postupgrade_health_report_for_vsc
-    vsc_username: "{{ vsc_custom_username | default(vsc_default_username) }}"
-    vsc_password: "{{ vsc_custom_password | default(vsc_default_password) }}"
-  roles:
-    - vsc-health
-
-- hosts: primary_vsds
-  gather_facts: no
-  pre_tasks:
-    - name: Set upgrade flag
-      set_fact:
-        nuage_upgrade: true
-
-    - name: Check if prereq satisfied for upgrade
-      include_role:
-        name: common
-        tasks_from: check-upgrade-prereq
-  vars:
-    report_filename: vstat_postupgrade_health_report_for_vsd.txt
-  roles:
-    - vstat-vsd-health
-
-- hosts: vscs
-  gather_facts: no
-  pre_tasks:
-    - name: Set upgrade flag
-      set_fact:
-        nuage_upgrade: true
-
-    - name: Check if prereq satisfied for upgrade
-      include_role:
-        name: common
-        tasks_from: check-upgrade-prereq
-  vars:
-    report_filename: vstat_postupgrade_health_report_for_vsc.txt
-    vsc_username: "{{ vsc_custom_username | default(vsc_default_username) }}"
-    vsc_password: "{{ vsc_custom_password | default(vsc_default_password) }}"
-  roles:
-    - vstat-vsc-health
-
-- hosts: vrss
-  gather_facts: no
-  pre_tasks:
-    - name: Set upgrade flag
-      set_fact:
-        nuage_upgrade: true
-
-    - name: Check if prereq satisfied for upgrade
-      include_role:
-        name: common
-        tasks_from: check-upgrade-prereq
-  vars:
-    report_filename: vstat_postupgrade_health_report_for_vrs.txt
-  roles:
-    - vstat-vrs-health
+- name: Run Postupgrade health
+  import_playbook: "vsp_health.yml"
+  vars:
+    vsd_report_name: vsp_postupgrade_health_report_for_vsd
```

Comment:
Should we make all of these as .txt extentions for consistency?

Reply:
For the test I ran yesterday, here's what the reports look like on the gcp testbed:
`
[caso@testbed-4 ~]$ ls nuage-metroae/reports/
vsp_postupgrade_health_report_for_vsc-192.168.1.121.txt  vsp_preupgrade_health_report_for_vsd-192.168.1.111.txt
vsp_postupgrade_health_report_for_vsd-192.168.1.111.txt  vstat_postupgrade_health_report-192.168.1.131.txt
vsp_preupgrade_health_report_for_vsc-192.168.1.121.txt   vstat_preupgrade_health_report-192.168.1.131.txt
`

for those specific reports, the IP.txt part was appended somewhere outside the files I was working with. Since it appears to append a .txt to the file name, maybe it would be better to remove the .txt extension from some of the file names? I'm not sure if this is something that only occurs when working with gcp. The test I've been working with is a clone of upgrade-gcp-sa

Reply:
Okay, I see there is an underlying inconsistency in the application of .txt in the end for some files. Got it.